src/main/java/org/time2java/tRussianBanr/domain/User.java

```diff
@@ -7,4 +7,11 @@
     String name ;
     String vkID ;
     String city ;
+    Integer paymentNum ;
+    Integer paymentSum ;
+    Integer debtCount ;
+    Integer currentDeb ;
+    Integer earlyReturn ;
+    Integer delayReturn ;
```

Comment:
\[Checkstyle\] ERROR: Variable 'delayReturn' must be private and have accessor methods\.